src/avatar.jsx

```diff
@@ -86,6 +83,10 @@ const Avatar = React.createClass({
     this.setState({muiTheme: newMuiTheme});
   },
 
+  getTheme() {
```

Comment:
> It's not us it's javascript :laughing: :laughing: although nested destructuring takes some of the pain away

Haha! I agree about regarding the nested destructuring

@pdf @oliviertassinari Let's get rid of the getTheme function, looking again this is fairly simple component, and I think it's fine just breaking it up earlier into a `const` in the render method. Ironically this component could become a functional component once `style-propable` is gone.